app/views/crops/show.html.erb

```diff
@@ -81,7 +81,7 @@
   </div>
   <div data-equalizer-watch class="guide-results-container columns medium-6 small-12">
     <h2 class="text-center"><%= @guides.empty? ? "There are no guides for this crop yet" : "Guides for this crop" %></h2>
-    <%= link_to new_guide_path, :class => "add-guide" do %>
+    <%= link_to new_guide_path(:crop_id => @crop_id), :class => "add-guide" do %>
```

Comment:
should the second one here be `@crop.id`? (note the dot and not the underscore).